cert-renewer.Dockerfile

```diff
@@ -1,38 +1,39 @@
-FROM ubuntu:18.04
+FROM ubuntu:22.04
 
 # No interactive frontend during docker build
 ENV DEBIAN_FRONTEND=noninteractive \
     DEBCONF_NONINTERACTIVE_SEEN=true
 
+ENV WPT_HOST=wpt.live \
+  WPT_ALT_HOST=not-wpt.live \
+  WPT_BUCKET=wpt-live
+
+# Pin the versions for repeatable builds
+# For ubuntu package versions, go to https://packages.ubuntu.com/
+#   Search for the package with the "jammy" distribution (aka 22.04) selected.
+# For Google Cloud, look under https://packages.cloud.google.com/apt/dists/cloud-sdk/main/binary-amd64/Packages
 RUN apt-get -qqy update && \
   apt-get -qqy install \
-    apt-transport-https \
-    curl \
-    software-properties-common && \
-  CLOUD_SDK_REPO="cloud-sdk-$(grep VERSION_CODENAME /etc/os-release | cut -d '=' -f 2)" && \
-  echo "deb http://packages.cloud.google.com/apt $CLOUD_SDK_REPO main" | \
+    apt-transport-https=2.4.6 \
```

Comment:
Cool, I didn't know about this syntax! Is there something equivalent to Pipfile or package.json for apt, that can be updated with tools?

Reply:
I haven't used a tool like that for apt before. However, I just found this tool: https://github.com/Jille/dockpin

It creates a lock file. Might be worth looking into!